tests/e2e/parallel/main_test.go

```diff
@@ -2795,3 +2798,183 @@ func TestScheduledSuiteTimeoutFail(t *testing.T) {
 	}
 }
+
+func TestSuspendScanSetting(t *testing.T) {
+	t.Parallel()
+	f := framework.Global
+
+	// Creates a new `ScanSetting`, where the actual scan schedule doesn't necessarily matter, but `suspend` is set to `False`
+	scanSettingName := framework.GetObjNameFromTest(t) + "-scansetting"
+	scanSetting := compv1alpha1.ScanSetting{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      scanSettingName,
+			Namespace: f.OperatorNamespace,
+		},
+		ComplianceSuiteSettings: compv1alpha1.ComplianceSuiteSettings{
+			AutoApplyRemediations: false,
+			Schedule:              "0 1 * * *",
+			Suspend:               false,
+		},
+		Roles: []string{"master", "worker"},
+	}
+	if err := f.Client.Create(context.TODO(), &scanSetting, nil); err != nil {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), &scanSetting)
+
+	// Bind the new ScanSetting to a Profile
+	bindingName := framework.GetObjNameFromTest(t) + "-binding"
+	scanSettingBinding := compv1alpha1.ScanSettingBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      bindingName,
+			Namespace: f.OperatorNamespace,
+		},
+		Profiles: []compv1alpha1.NamedObjectReference{
+			{
+				Name:     "ocp4-cis",
+				Kind:     "Profile",
+				APIGroup: "compliance.openshift.io/v1alpha1",
+			},
+		},
+		SettingsRef: &compv1alpha1.NamedObjectReference{
+			Name:     scanSetting.Name,
+			Kind:     "ScanSetting",
+			APIGroup: "compliance.openshift.io/v1alpha1",
+		},
+	}
+	if err := f.Client.Create(context.TODO(), &scanSettingBinding, nil); err != nil {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), &scanSettingBinding)
+
+	// Wait until the first scan completes since the CronJob is created
+	// after the scan is done
+	if err := f.WaitForSuiteScansStatus(f.OperatorNamespace, bindingName, compv1alpha1.PhaseDone, compv1alpha1.ResultNonCompliant); err != nil {
+		t.Fatal(err)
+	}
+
+	// Assert the ComplianceSuite.suspend attribute is False
+	suite := &compv1alpha1.ComplianceSuite{}
+	key := types.NamespacedName{Name: bindingName, Namespace: f.OperatorNamespace}
+	if err := f.Client.Get(context.TODO(), key, suite); err != nil {
+		t.Fatal(err)
+	}
+	if suite.Spec.Suspend == true {
+		t.Fatalf("ComplianceSuite %s is suspended when it should not be", suite.Name)
+	}
+
+	// Assert the CronJob associated with the ComplianceSuite is False
+	job := &batchv1.CronJob{}
+	jobName := compsuitectrl.GetRerunnerName(suite.Name)
+	err := f.WaitForObjectToExist(jobName, f.OperatorNamespace, job)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if *job.Spec.Suspend == true {
+		t.Fatalf("CronJob %s is suspended when it should not be", jobName)
+	}
+
+	// Suspend the `ScanSetting` using the `suspend` attribute
+	scanSettingUpdate := &compv1alpha1.ScanSetting{}
+	if err := f.Client.Get(context.TODO(), types.NamespacedName{Namespace: f.OperatorNamespace, Name: scanSettingName}, scanSettingUpdate); err != nil {
+		t.Fatalf("failed to get ScanSetting %s", scanSettingName)
+	}
+	scanSettingUpdate.Suspend = true
```

Comment:
For some reason in my local testing, this isn't actually updating the `CronJob`.